scripts/setup-flow.sh

```diff
@@ -1,4 +1,8 @@
 # copy config files to design dir
-rsync --archive designs/${design}/. ${design_dir}
+if [ -z "$NICKNAME" ]; then
```

Comment:
@vvbandeira  
Locally tested its working fine with command: ./scripts/run-design.sh wb_host mbist_ctrl
In CI its got failed during rsync. Can you please review this and why its got failed.
Thanks

Reply:
Why not use `rsync --archive designs/mbist_ctrl/. ${design_dir}` and remove the if statement?